exercises/guillaume_communie/ex01_basics/src/main.cpp

```diff
@@ -0,0 +1,93 @@
+/**
+ * Skeleton main routine
+ */
+
+#include <iostream>
+#include <fstream>
+#include <map>
+#include <string>
+#include <cctype>
+#include <iomanip>
+#include <set>
+
+const std::string RESULTS_FILENAME("word_counts.data");
+
+/**
+ * Print the program usage.
+ */
+void usage(void) {
+  std::cout << "Usage: WordCounter <file_name>" << std::endl;
+  exit(1);
+}
+
+/**
+ * Check if a characters is an 'interword' character.
+ * @param[in] c The character to test
+ * @return      True if the character is an interword character
+ */
+bool isInterWord(int c) {
```

Comment:
You can store these symbols in a string, then check if you can find the char in it, this might be nicer.